tests/libs/util/wer_report.hpp

```diff
@@ -50,14 +50,24 @@ class _wer_report
     static constexpr const char environment_variable_name[] = "EBPF_ENABLE_WER_REPORT";
     static constexpr const char environment_variable_value[] = "yes";
     static constexpr const wchar_t wer_event_type[] = L"Test Application Crash";
+
     static constexpr unsigned long fatal_exceptions[] = {
-        EXCEPTION_ILLEGAL_INSTRUCTION,
-        EXCEPTION_STACK_OVERFLOW,
-        EXCEPTION_ACCESS_VIOLATION,
-        EXCEPTION_INT_DIVIDE_BY_ZERO,
-        STATUS_HEAP_CORRUPTION,
+        STATUS_ACCESS_VIOLATION,
         STATUS_ASSERTION_FAILURE,
-        STATUS_BREAKPOINT};
+        STATUS_BREAKPOINT,
+        STATUS_DATATYPE_MISALIGNMENT,
+        STATUS_GUARD_PAGE_VIOLATION,
+        0xC0000235L, // STATUS_HANDLE_NOT_CLOSABLE
```

Comment:
why is there no define?  maybe add a comment explaining why you need the integer literal?